app/services/big_blue_button_api.rb

```diff
@@ -12,13 +12,57 @@ def bbb_server
     @bbb_server ||= BigBlueButton::BigBlueButtonApi.new(bbb_endpoint, bbb_secret, '1.8')
   end
 
+  # Start a session for a specific room and returns the join URL.
+  def start_session(room:, session_starter:, options: {})
+    # TODO: amir - Revisit this.
+    create_options = default_create_opts.merge(options)
+    join_options = default_join_opts.merge(create_options) # TODO: amir - Revisit this (createTime,...).
+
+    meeting_id = room.friendly_id
+    user_name = session_starter&.name || 'Someone'
+    password = (session_starter && create_options[:moderatorPW]) || create_options[:attendeePW]
```

Comment:
We don't actually need the password anymore - I think there's a new param called `role` set to either `viewer` or `moderator`

Reply:
Didn't know about it, I'll check the BBB API and use the new approach.

Reply:
Unfortunately, it appears that there's some problems with using `role` parameter.
I'll show you the issue, for now I'll keep it this way until we figure out what to do.